.travis.yml

```diff
@@ -1,7 +1,24 @@
 services:
 - docker
+
+before_script:
+  - docker login -u $DOCKER_HUB_USER -p $DOCKER_HUB_PASSWORD   #login to dockerhub
```

Comment:
It's generally recommended to pass the password by piping it through std-in to prevent it from being logged in any history - ([see the docker docs](https://docs.docker.com/engine/reference/commandline/login/), in our case we can just `echo $DOCKER_HUB_PASSWORD | docker login ... `)